googleSchema.js

```diff
@@ -0,0 +1,55 @@
+// Philosophy FAQ
+// to add more philosophy questions just add them in this json as the first 2 and the script will do the rest
+var faqPhilosophy = {
+	"faqs": {
+		"protectingOurDemographics": {
+			"question": "What does Destiny think about protecting our demographics?",
+			"answer": "Destiny believes that this endeavor is futile because he rejects the notion that there is a consistent and coherent definition of American values",
+		},
+		"protestingRioting": {
+			"question": "What does Destiny think about the protesting and rioting?",
+			"answer": "Destiny ardently believes in people's right to protest and will always support the right to protest, even if he doesn't necessarily agree with the positions being advocated for by any particular protest. ",
+		},
+	}
+};
+
+// Initialize JSON array and data
+var faqsPhilosophy = faqPhilosophy.faqs;
+faqsPhilosophy = JSON.stringify(faqsPhilosophy);
```

Comment:
don't do this

Reply:
I changed the way this all worked, before I would take an array and transform it to string then parse it with JSON.parse. Now it's a string from the start, and it gets parsed by json parsed only. This is a video made by google suggesting this approach of using string rather than instantly object https://www.youtube.com/watch?v=ff4fgQxPaO0&